docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
+
+The ``deploy_post`` task, which is part of the ``config_dev``, ``config_qa``, and ``config_managed`` flows, is responsible for deploying these bundles.
+
+.. important:: Do not include metadata in ``unpackaged/post`` unless it is intended to be delivered to *all* environments (both managed installations and unmanaged deployments). It's also critical for managed package projects that this metadata include namespace tokens (see `namespace injection`_).
+
+
+``unpackaged/config``: Tailor an Org
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Projects can come with more than one supported configuration in their CumulusCI automation.
+
+    Example: Projects often support distinct, tailored ``dev_org``, ``qa_org``, and ``install_prod`` flows, each of which performs a unique setup for their specific use case.
+
+Unpackaged metadata stored in ``unpackaged/config`` is a tool to support operational needs that tailor orgs to different configurations. 
+
+    Example: A testing-oriented scratch org needs to deploy a customized set of Page Layouts to help testers easily visualize data under test. Such page layouts are stored in ``unpackaged/config/qa``.
+
+
+
+Unpackaged Metadata Folder Structure
+------------------------------------
+
+All unpackaged metadata is stored in the ``unpackaged`` directory tree, which contains these top-level directories.
+
+* ``unpackaged/pre``
+* ``unpackaged/post``
+* ``unpackaged/config``
+
+These trees contain metadata bundles in Metadata API format, represented as a directory containing a ``package.xml`` manifest and Metadata API-format source code. CumulusCI does not support Salesforce DX format for unpackaged bundles.
+
+
+
+Namespace Injection
 -------------------
 
-Customize config flows
+Projects that build managed packages often construct their unpackaged metadata to be deployable in multiple contexts.
+
+* Unmanaged deployments, such as developer orgs
+* Unmanaged namespaced scratch orgs
+* Managed contexts, such as a beta test org or a demo org created with ``install_prod``
+
+Because projects that are building an org implementation or a non-namespaced package do not have a namespace, or a distinction between managed and unmanaged contexts, these projects must also build namespaced packages.
+
+    Example: Metadata located in ``unpackaged/post`` is deployed after the application code in both unmanaged and managed contexts. If that metadata contains references to the application components, it must be deployable when that metadata is nampespaced (in a managed context or namespaced scratch org) *and* when it is not (in an unmanaged context).
+
+CumulusCI uses a strategy called *namespace injection* to support this use case. Namespace injection is very powerful, and requires care from application implementors to ensure that metadata remains deployable in all contexts.
```

Comment:
"application implementors"

Just confirming this is what we want to call them because this is the first time I've heard this label being used.

Reply:
We could maybe use "application developers and architects" as a substitute?

Reply:
```suggestion
CumulusCI uses a strategy called *namespace injection* to support this use case. Namespace injection is very powerful, and requires care from the project team to ensure that metadata remains deployable in all contexts.
```